src/SignalR/clients/ts/signalr/src/HttpConnection.ts

```diff
@@ -608,7 +608,7 @@ export class TransportSendQueue {
             offset += item.byteLength;
         }
 
-        return result;
+        return result.buffer;
```

Comment:
? who uses this?

Reply:
The `getDataDetail` method which prints the info I put in the sumary

Reply:
Interesting that the type checker didn't catch this.

Reply:
ArrayBuffer is a weird type. It has basically no public members, you have to wrap it in a view like `Uint8Array`. It's possible that, because of that, the `result` object technically does adhere to the `ArrayBuffer` interface (structurally, since that's all TS actually cares about).

Short answer: Type checker be weird